make.ps1

```diff
@@ -232,16 +232,16 @@ if ($args.Length -eq 0)
 {
 	echo "Command list:"
 	echo ""
-	echo "  all             Builds the game and its development tools."
-	echo "  dependencies    Copies the game's dependencies into the main game folder."
-	echo "  version         Sets the version strings for the default mods to the latest"
-	echo "                  version for the current Git branch."
-	echo "  clean           Removes all built and copied files. Use the 'all' and"
-	echo "                  'dependencies' commands to restore removed files."
-	echo "  test            Tests the default mods for errors."
-	echo "  check           Checks .cs files for StyleCop violations."
-	echo "  check-scripts   Checks .lua files for syntax errors."
-	echo "  docs            Generates the trait and Lua API documentation."
+	echo "  all, a              Builds the game and its development tools."
+	echo "  dependencies, d     Copies the game's dependencies into the main game folder."
+	echo "  version, v          Sets the version strings for the default mods to the latest"
```

Comment:
![image](https://user-images.githubusercontent.com/3399086/37241773-b744fb30-2456-11e8-80ba-d9a96eb0f8ff.png)

Can you wrap latest? It's 1 character too long on the default size.

Reply:
"Sets the version strings for the default mods to the latest" reads weirdly anyway, so how about shortening it to "Sets the version strings for the default mods"?

Reply:
I've just moved the last word into the next line, couse it's a twoliner anyway. Could be changed if still wanted.
Commits squashed